### PR TITLE
docs(readme): Update README.md with more detailed install instructions

### LINKED ISSUE #145: How to integrate "bootstrap-vue" to "vue-cli webpack template" and using?
Can you clearly explain clearly in the way to integrate both of them, and the ways of using bootstrap for the beginner?

### PR DESCRIPTION
**Pull request by kylemh**
Resolve #145 

"Can you clearly explain clearly in the way to integrate both of them, and the ways of using bootstrap for the beginner?"

Instructions updated and written to be more detailed for users of various Vue project structures (or vue-cli templates).

**Screenshot Preview:**

Preview of README.md in Preview view of a GitHub post.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,25 +34,49 @@
...
+import BootstrapVue from 'bootstrap-vue/dist/bootstrap-vue.esm';
+import 'bootstrap-vue/dist/bootstrap-vue.css';
+import 'bootstrap/dist/css/bootstrap.css';
+/* ( there may be other imports here ) */
+
+Vue.use(BootstrapVue);
+```
+
+#### Note on style-loader:
+If you are unable or do not want to add style-loader as a developer dependency, you have to manually include both [Bootstrap's](https://v4-alpha.getbootstrap.com/getting-started/download/) and [BootstrapVue's](https://unpkg.com/bootstrap-vue@0.18.0/dist/bootstrap-vue.css) CSS files in your bundle or reference them from `static/` via `index.html`.
```
Comment (tmorehouse) [inline README.md]: To make this revision independant, is would suggest changin 
```
[BootstrapVue's](https://unpkg.com/bootstrap-vue@0.18.0/dist/bootstrap-vue.css)
```

To
```
[BootstrapVue's](https://unpkg.com/bootstrap-vue@latest/dist/bootstrap-vue.css)
```

Comment (kylemh) [inline README.md]: Good call - will push fix.

**Comment by kylemh** [comment]
Shucks - I just realized it should be:

`yarn add bootstrap@4.0.0-alpha.6` - not just bootstrap. Apologies.

**Comment by tmorehouse** [comment]
The changes have been merged.

Thanks!

**Pull request merged**